app/models/user.rb

```diff
@@ -7,4 +7,11 @@ class User < ActiveRecord::Base
   validates :username,
     presence: true,
     uniqueness: true
+
+  def self.genders
+    [["Not specified", "Not specified"],
+     ["Male", "Male"],
+     ["Female", "Female"],
+     ["Other", "Other"]]
+  end
```

Comment:
Instead of a method `self.genders`, you might take advantage of using a constant. 

```ruby
class User < ActiveRecord::Base
  GENDERS = [
    ["Not specified", "Not specified"],
    ["Male", "Male"],
    ["Female", "Female"],
  ]

  # Continued...
end
```

In the view, I would call the constant this way:

```ruby
User::GENDERS
```